### PR TITLE
Confirm following from banner

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

If there are inactive neurons, the user should be able to reset their voting-power-refreshed timestamp. In this PR, we add a ["Confirm Following" button](https://github.com/dfinity/nns-dapp/pull/5990) to the modal that displays the user’s inactive neurons.

# Changes

- Add the `ConfirmFollowingButton` to the modal.
- Close the modal when all inactive neurons are refreshed.

# Tests

- Added.

# Todos

- [ ] Add entry to changelog (if necessary).
Not necessary.

### CODE REVIEW

File: frontend/src/tests/lib/components/neurons/LosingRewardNeuronsModal.spec.ts [hunk 1]
```diff
@@ -109,6 +129,30 @@ describe("LosingRewardNeuronsModal", () => {
...
   });
 
+  it("should confirm following", async () => {
+    neuronsStore.setNeurons({
+      neurons: [in10DaysLosingRewardsNeuron, losingRewardsNeuron],
+      certified: true,
+    });
+    const po = await renderComponent({});
+
+    expect((await po.getNnsLosingRewardsNeuronCardPos()).length).toEqual(2);
```
Comment (yhabib) [inline frontend/src/tests/lib/components/neurons/LosingRewardNeuronsModal.spec.ts:139]: there was a typo when creating `getNnsLosingRewardsNeuronCardPos`, it shouldn't ahve the last `s`, no?

File: frontend/src/tests/lib/components/neurons/LosingRewardsBanner.spec.ts [hunk 2]
```diff
@@ -109,4 +135,48 @@ describe("LosingRewardsBanner", () => {
...
+    const po = await renderComponent();
+
+    await po.clickConfirm();
+    await runResolvedPromises();
+    await po.getLosingRewardNeuronsModalPo().clickConfirmFollowing();
+    await runResolvedPromises();
+
+    expect(spyRefreshVotingPower).toBeCalledTimes(1);
+    expect(spyConsoleError).toBeCalledTimes(1);
+    expect(await po.getLosingRewardNeuronsModalPo().isPresent()).toEqual(true);
```
Comment (yhabib) [inline frontend/src/tests/lib/components/neurons/LosingRewardsBanner.spec.ts:180]: dont you need to advance the clock like [before](https://github.com/dfinity/nns-dapp/pull/6004/files#diff-fa9ba4a45fdb3fa22b00282032014a79db977e16348cc889273700afdf271ef5R156)?
```
    vi.advanceTimersToNextFrame();
```

File: frontend/src/tests/lib/components/neurons/LosingRewardNeuronsModal.spec.ts (hunk 1 above)
Comment (mstrasinskis) [reply frontend/src/tests/lib/components/neurons/LosingRewardNeuronsModal.spec.ts:139]: If you mean the “s” in “Pos,” it’s there intentionally as the plural of page object.

File: frontend/src/tests/lib/components/neurons/LosingRewardsBanner.spec.ts (hunk 2 above)
Comment (mstrasinskis) [reply frontend/src/tests/lib/components/neurons/LosingRewardsBanner.spec.ts:181]: Good point, added.

**Pull request merged**